homeassistant/components/iaqualink/binary_sensor.py

```diff
@@ -41,7 +42,7 @@ def is_on(self) -> bool:
         """Return whether the binary sensor is on or not."""
         return self.dev.is_on
```

Comment:
This one can be done in the constructor, needs a separate PR

Reply:
This is a dynamic attribute right?

Reply:
#100281